cmd/fyne/version.go

```diff
@@ -0,0 +1,74 @@
+package main
+
+import (
+	"fmt"
+	"go/build"
+	"os"
+	"strings"
+
+	"github.com/pkg/errors"
+)
+
+const pkg = "fyne.io/fyne"
+
+// Declare conformity to command interface
+var _ command = (*versioner)(nil)
+
+type versioner struct {
+	context build.Context
+}
+
+func (v *versioner) run(_ []string) {
+	err := v.validate()
+	if err != nil {
+		fmt.Fprintln(os.Stderr, err.Error())
+		os.Exit(1)
+	}
+
+	err = v.doVersion()
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "%s\n", err.Error())
+		os.Exit(1)
+	}
+}
+
+func (v *versioner) validate() error {
```

Comment:
Why do we need a method that does nothing? The call to it above also seems un-needed.

Reply:
That's my bad.  It was in the other commands, so I thought it was a required method on the `command` interface; I should have checked.